cas/ctg_store.c

```diff
@@ -966,6 +966,10 @@ static void ctg_store_release(struct m0_ref *ref)
 
 	M0_ENTRY();
 	m0_mutex_fini(&ctg_store->cs_state_mutex);
+	// TODO: Cleanup all in-memory allocations done for indices in meta tree
+	ctg_fini(ctg_store->cs_state->cs_meta);
+	ctg_fini(ctg_store->cs_ctidx);
+	ctg_fini(ctg_store->cs_dead_index);
 	ctg_store->cs_state = NULL;
```

Comment:
Here we are trying to assign NULL value to ctg_store->cs_state without free the pointer. 
I think, we have to free the memory before assign NULL value.

Reply:
There is no allocation using m0_alloc for ctg_store->cs_state.
Hence no need for m0_free()